.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thank you again for  pointing that out, should be included in my global gitignore.